tests/countries/test_bosnia_and_herzegovina.py

```
@@ -118,113 +107,130 @@ def test_independence_day(self):
         name = "Uskrsni ponedjeljak (Katolički)"
-        dt = (
+        dts = (
             "2020-04-13",
             "2021-04-05",
             "2022-04-18",
             "2023-04-10",
         )
-        self.assertHolidayName(name, dt)
+        self.assertHolidayName(name, dts)
+        self.assertHolidayName(name, self.full_range)
         for subdiv, holidays in self.subdiv_holidays.items():
```

Ruff (0.14.3)

Loop control variable `subdiv` not used within loop body

Rename unused `subdiv` to `_subdiv`

(B007)